problems/sep_3_2018.js

```diff
@@ -8,8 +8,11 @@
 */
 
 function LongestWord(sen) {
-  // code goes here
-  return sen;
+  const arrSen = sen.split(' ')
+
+  return arrSen
+    .map((str) => str.replace(/[^a-zA-Z0-9]+/g, ''))
+    .reduce((curr, next) => curr.length >= next.length ? curr : next) // prettier-ignore
```

Comment:
i know what the `// prettier-ignore` comment is for.. but i dont necessarily agree with it. for someone that doesn't know prettier, they may not know what you mean and it has nothing to do with the code either. maybe just disable prettier on save